widget/gridwrap.go

```diff
@@ -179,6 +199,70 @@ func (l *GridWrap) ScrollToOffset(offset float32) {
 	l.offsetUpdated(l.scroller.Offset)
 }
 
+// TypedKey is called if a key event happens while this GridWrap is focused.
+//
+// Implements: fyne.Focusable
+func (l *GridWrap) TypedKey(event *fyne.KeyEvent) {
+	switch event.Name {
+	case fyne.KeySpace:
+		l.Select(l.currentFocus)
+	case fyne.KeyDown:
+		count := 0
+		if f := l.Length; f != nil {
+			count = f()
+		}
+		// TODO l.RefreshItem(l.currentFocus)
+		l.currentFocus += l.getColCount()
+		if l.currentFocus >= count-1 {
+			l.currentFocus = count - 1
+		}
+		l.scrollTo(l.currentFocus)
+		l.Refresh() // TODO l.RefreshItem(l.currentFocus)
```

Comment:
Not a blocker for this PR, but I think we should get the RefreshItem API in for GridWrap before 2.4 if possible

Reply:
Ah yes sorry - it is a separate PR that Tilo was looking at - hence the TODO here.
If we get to next week and it's still missing then I will go ahead and complete that aspect separately.